CMakeLists.txt

```diff
@@ -512,13 +512,17 @@ endif()
 option(WITH_SYSTEM_BOOST "require and build with system Boost" OFF)
 
 set(BOOST_COMPONENTS
-	thread system regex random program_options date_time iostreams coroutine context)
+	thread system regex random program_options date_time iostreams)
 set(BOOST_HEADER_COMPONENTS container)
 
 if(WITH_MGR)
 	list(APPEND BOOST_COMPONENTS python)
 endif()
 
+if(WITH_RADOSGW_BEAST_FRONTEND)
```

Comment:
we should disable `WITH_RADOSGW_BEAST_FRONTEND` archs not listed below :
- arm32
- aarch64
- i386
- amd64
- powerpc32
- powerpc64
- mips32

i am consulting `asm_context_sources` in `boost/libs/context/build/Jamfile.v2` for the above list.

in other words, following archs are not supported:

- alpha
- hppa
- ia64
- m68k
- mips64
- mips64el
- s390
- s390x
- sh4
- sparc
- sparc64
- x32

i am consulting `boost1.63-1.63.0+dfsg/debian/rules` for the list above.

considering the length of these lists and their authorities, probably we should use the former.